src/main/java/edu/unifi/controller/HomeController.java

```diff
@@ -20,4 +22,6 @@ public List<Room> getRooms() {
     public Room getById(String name) {
         return RoomDAO.getInstance().getById(name);
     }
+
+    public User getUserRoleByUsername(String name) { return UserDAO.getInstance().getByUsername(name); }
```

Comment:
method definition is misleading, if the name is "getUserRole" you should return a Role